modules/actions/pool/v3/sync-hook-data.ts

```diff
@@ -29,16 +30,27 @@ export const syncHookData = async (
             continue;
         }
         const keys = Object.keys(hooksTypes) as HookType[];
-        const hookType = keys.find((key) => hooksTypes[key]?.includes(hookData.address));
+        const hookKey = keys.find((key) => hooksTypes[key]?.includes(hookData.address));
+        let hookType: GqlHookType = 'UNKNOWN';
+        switch (hookKey) {
+            case 'feeTakingHook':
+                hookType = 'FEE_TAKING';
+            case 'exitFeeHook':
+                hookType = 'EXIT_FEE';
+            case 'stableSurgeHook':
+                hookType = 'STABLE_SURGE';
+            default:
+                hookType = 'UNKNOWN';
+        }
```

Comment:
is the main reason to have the graphql response typed? looks like types are the same as names were, because names are set by us, at the end it's just a different string. maybe let's change the names to these uppercase ones, so we can skip mapping name to type?